pkg/api/networkservice/mechanisms/kernel/helpers.go

```diff
@@ -57,3 +62,11 @@ func (m *mechanism) GetNetNSInode() string {
 	}
 	return m.GetParameters()[common.NetNSInodeKey]
 }
+
+// GetPCIAddress returns PCI address of the VF device
+func (m *mechanism) GetPCIAddress() string {
+	if m == nil || m.GetParameters() == nil {
+		return ""
```

Comment:
Can we return a constant here? Like `"KERNEL"`, meaning that the device is not a physical one (incliuding SR-IOV VFs), but rather a pure kernel - tun/tap, veth, macvlan etc.

Reply:
I'd rather have a method like IsPCIDevice() that would properly indicate whether we are or are not a PCIDevice.

Reply:
Could we simply return nil here?

Reply:
@rktidwell In Golang, a type has a 'zero value' ... for pointers its nil, for strings it "".  So returning "" is sort of the string equivalent of returning nil.

All of that said, @nickolaev What do you think of my IsPCIDevice() bool suggestion?

Reply:
I agree. Let's go with your proposal.